src/components/Pagination.vue

```diff
@@ -76,14 +72,21 @@ export default {
 
   //Functions for navigating between page questions
   methods: {
-    nextPage() {
-      const routeId = this.$route.params.id++;
-      // Unary Operator By adding a + sign before a String, it will be converted into a number if it follows the right format.
-      // https://stackabuse.com/javascript-convert-string-to-number/
-      if (+this.$route.params.id > this.questions.length) {
-        return this.$router.push('/results');
+    nextPage() {  
+      const routeId = this.$route.params.id;
+      const nextRouteId = +routeId + 1;
+
+      let newRoute = { 
+        params: {
+          id: nextRouteId, 
+        },
+      };
+
+      if(+routeId === this.questions.length) {
+        newRoute = '/results';
```

Comment:
```suggestion
    nextPage() {  
      const routeId = +this.$route.params.id;
      const nextRouteId = routeId + 1;

      let newRoute = { 
        params: {
          id: nextRouteId, 
        },
      };

      if(routeId === this.questions.length) {
        newRoute = '/results';
```

The `+` could be moved to the `routeId` constant to avoid duplicating it